src/pages/helpcase-profile/[residentId]/index.js

```diff
@@ -51,12 +51,25 @@ export default function HelpcaseProfile({ residentId }) {
                         categorisedCaseNotes[hr.helpNeeded].push(caseNoteObject[0])
                         categorisedCaseNotes['All'].push(caseNoteObject[0])
                     }
-                    categorisedCaseNotes.All.sort((a, b) => new Date(b.noteDate) - new Date(a.noteDate))
-                    categorisedCaseNotes['Welfare Call'].sort((a, b) => new Date(b.noteDate) - new Date(a.noteDate))
-                    categorisedCaseNotes['Help Request'].sort((a, b) => new Date(b.noteDate) - new Date(a.noteDate))
-                    categorisedCaseNotes['CEV'].sort((a, b) => new Date(b.noteDate) - new Date(a.noteDate))
-                    categorisedCaseNotes['Contact Tracing'].sort((a, b) => new Date(b.noteDate) - new Date(a.noteDate))
                 }
+                else if(!isJSON(hr.caseNotes)){
+                    let nonJsonCaseNotesArray = getNonJsonCasenotesArray(hr.caseNotes)
+                    nonJsonCaseNotesArray.forEach(nonJsonCaseNote => {
+                        if (nonJsonCaseNote) {
+                            let caseNoteObject = {"author": getAuthor(nonJsonCaseNote),
+                                                        "formattedDate": formatDate(getDate(nonJsonCaseNote)),
+                                                        "note": getNote(nonJsonCaseNote),
+                                                        "helpNeeded":hr.helpNeeded,
+                                                        "noteDate": getDate(nonJsonCaseNote)
+                                                        }
+                            categorisedCaseNotes[hr.helpNeeded].push(caseNoteObject) 
+                            categorisedCaseNotes.All.push(caseNoteObject)
+                        }
+                    });
+                }
+                helpTypes.forEach(helpType => {
+                    categorisedCaseNotes[helpType].sort((a, b) => new Date(b.noteDate) - new Date(a.noteDate))
+                });
```

Comment:
I'd probably move this into gateway. We will probably want to change the way we return case notes from the api in the future (so that we don't have any more json case notes) and it'd be easier to only change that in the GW

Reply:
I agree with this, i will release this now and make this refactor with the next piece of work